src/style.css

```diff
@@ -1 +1,58 @@
 /* Styles go here */
+:root {
+  --blue: #00acdc;
+  --black: #000;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  padding: 0 50px;
+  justify-content: space-between;
+}
+
+.logo_img {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  display: inline-block;
+  position: relative;
+  text-decoration: none;
+  font-family: "Roboto", sans-serif;
+  color: var(--black);
+  font-size: 12px;
+  line-height: 14px;
+  font-weight: 500;
+  text-transform: uppercase;
+  padding: 23px 0;
+}
+
+.nav__link:hover,
```

Comment:
There is no need to change a position for hovered elements, consider applying just a color change here

Reply:
Hi, you mean I don't need the row 51 position: absolute; I can delete him?